admin/test_config.js

```diff
@@ -111,11 +110,6 @@ module.exports = function (config, callback) {
       tlsOptions = {
         ca: cas
       };
-
-      if (nconf.get('SSL_ENABLE_EMPTY_SUBJECT')) {
```

Comment:
please remove all references to SSL_ENABLE_EMPTY_SUBJECT and `lib/tls`

Reply:
done.